src/App.js

```diff
@@ -290,7 +294,11 @@ function InputDamjiroGakufu ({ dispatch }) {
             const videoId = json.youtubeVideoId
             const timeOffset = json.timeOffset
             if (!isString(videoId) || !isNumber(timeOffset))
-              throw new Error('Invalid JSON')
+              dispatch({
+                type: 'SNACK_LOAD',
+                message: 'invalid JSON',
+                variant: 'error'
+              })
```

Comment:
This call of `dispatch` won't interrupt the process, so its following lines (L.302~304) will be executed, which is not correct.
Probably `throw new Error(...)` should be untouched here, and dispatch `SNACK_LOAD` in the `catch` clause (L.306).

Reply:
>  so its following lines (L.302~304) will be executed, which is not correct.
I see, fix it. However, throwing a new error is some difficulty. See below.

Reply:
Fix it to move following lines in else block.

Reply:
Please wrap these lines in `{}`.